pom.xml

```diff
@@ -28,6 +28,7 @@
 
     <!--Folio dependencies properties-->
     <folio-module-descriptor-validator.version>1.0.0</folio-module-descriptor-validator.version>
+    <folio-kafka-wrapper.version>3.1.1</folio-kafka-wrapper.version>
```

Comment:
3.2.0 can be used as the freshest version

Reply:
Thanks, updated